makefiles/Makefile.inc

```diff
@@ -29,6 +29,11 @@
 
 # Backwards-compatibility wrapper for people using
 # "include $COCOTB/makefiles/Makefile.inc" in their Makefile.
+
+$(warning 'Deprecated cocotb file structure detected. Please install cocotb \
+(see documentation) and set "COCOTB:=$$(shell cocotb-config --share)" \
+or "include $$(shell cocotb-config --makefile)/Makefile.sim" in Makefiles')
+
 COCOTB_INSTALL_METHOD = srctree
 export COCOTB_INSTALL_METHOD
```

Comment:
Is this used anywhere?

Reply:
I can not see this anywhere. I think added by @imphil